client/my-sites/stats/features/modules/shared/stats-empty-action-email.tsx

```diff
@@ -35,9 +41,13 @@ const StatsEmptyActionEmail: React.FC< StatsEmptyActionProps > = ( { from } ) =>
 
 				// If the site is a Jetpack site, use the Jetpack links.
 				// Otherwise, use the WordPress.com links.
-				const redirectUrl = isJetpack ? JETPACK_SUPPORT_NEWSLETTER_URL : NEWSLETTER_SUPPORT_URL;
 
-				setShowSupportDoc( redirectUrl );
+				if ( isJetpack ) {
+					// open in new tab
+					setTimeout( () => window.open( supportLink, '_blank' ), 250 );
```

Comment:
Can you add this inside `useSupportDocData` so it's fixed for all links?

@alshakero did it [here](https://github.com/Automattic/wp-calypso/pull/104211/commits/70c541dfa9d6d4d0f4b1f08a6c4fe4dbaa9cae8b#diff-0807bc386e5be8e1486d275e7ac6a539a14c8616e8de3d7e0762a2fe03720bedR60), but the PR has not yet been merged.

Reply:
@mmtr looks like it's more complicated according to @alshakero slack's message :D

Reply:
Just out of curiosity, why the timeout? Won't it trigger the popup blocker?

Reply:
@alshakero that's what we had before the helpcenter changes I've made. I simply kept it :)